compilers/4.03.0/4.03.0+flambda-fs/4.03.0+flambda-fs.comp

```diff
@@ -0,0 +1,15 @@
+opam-version: "1"
+version: "4.03.0"
+src: "https://github.com/chambart/ocaml-1/archive/flambda+function_section.tar.gz"
+build: [
+  ["./configure" "-prefix" prefix "-with-debug-runtime"]
```

Comment:
we probably don't need/want the debug-runtime option here, if the goal is to have small executable. @chambart what do you recommend?